brains/andalite.py

```diff
@@ -79,18 +90,21 @@ def threadMain(self):
 
     async def messageConsumer(self, websocket):
         while True:
-            message = await websocket.recv()
-            data = ast.literal_eval(message)
-            print("received data:")
-            pprint.pprint(data)
-            self.inboundMemoryWrites.put_nowait(data)
+            messageString = await websocket.recv()
+            message = ast.literal_eval(messageString)
+            num = message.get('event_number', -1)
+            if num > self.currentEventNumber:
```

Comment:
if it's more than 1 bigger, we might wanna log something as well...  since we'll be skipping events and ignoring them if they do come in later.  eventually we'll want a "catch-up" path.  we might also want a buffer of some small number (3?) of events in case a few come in out of order, before resorting to skipping.

Reply:
alternatively we might decide to make the IGameSync unconditionally include *every* strategy's memory blocks in each update when any of them have changed, so skipping events is OK but the overall bandwidth is higher